### PR TITLE
Bug/Issue-379-ContestBase.Name-required

### PR DESCRIPTION
**Pull request by afsmythe**
Updates ContestBase.Name to be required. Additional housekeeping to remove obsolete CSV documentation files and format InternationalizedText page.

### CODE REVIEW

**Comment by cjerdonek** [comment]
I have a suggestion. When submitting changes that result in updates to the auto-generated files, I think it would be good to put the step of running the script to auto-generate the files in a separate commit (but in the same PR). That way it becomes easy to see the code changes independent of the changes to the auto-generated files (which there are a lot of). Currently, it hard to find the code changes in GitHub's UI because there are so many files.

**Comment by afsmythe** [comment]
Thanks @cjerdonek. To confirm do you mean to separate out the RST files from the rest of the update? So, in this case there would be one commit for the YAML update, another to include the updated RST files, and maybe another to summarize the deleted CSV items?

**Comment by cjerdonek** [comment]
Hi, @afsmythe. Yeah, I just meant keeping the "manual" changes (in this case YAML file) in commits separate from running the script (RST and CSV files). Separating the RST from CSV in commits isn't as important to me. If it's a massive file deletion done manually, I think it would be good to keep that separate, too (not sure if the CSV was manual or auto-generated in this case).

**Comment by afsmythe** [comment]
Sounds good. Will update the [Contributing Doc](https://github.com/votinginfoproject/vip-specification/blob/vip52/CONTRIBUTING.md) to add a step about separating out commits in this way.

**Comment by afsmythe** [comment]
@jdmgoogle just need a quick approval and I'll merge these in. Thanks!

File: docs/built_rst/csv/elements/polling_location.rst [hunk 1]
```diff
@@ -11,11 +11,11 @@ The PollingLocation object represents a site where voters cast or drop off ballo
...
 | address_line    | ``xs:string``            | **Required** | Repeats      | Represents the various parts of an       | At least one valid ``AddressLine`` must  |
-|                 |                          |              |              | address to a polling location. For CSV   | be present for ``PollingLocation`` to be |
-|                 |                          |              |              | implementations, repeating               | valid. If no valid ``AddressLine`` is    |
-|                 |                          |              |              | ``address_line`` columns should be       | present, the implementation is required  |
-|                 |                          |              |              | incremented, e.g. address_line1,         | to ignore the ``PollingLocation``        |
-|                 |                          |              |              | address_line2, address_line3.            | element containing it.                   |
+|                 |                          |              |              | address to a polling location.           | be present for ``PollingLocation`` to be |
```
Comment (jdmgoogle) [inline docs/built_rst/csv/elements/polling_location.rst]: Why were the address_line documentation notes removed?

File: docs/built_rst/tables/elements/polling_location.rst [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
-|                  |                                         |              |              | address to a polling location. For CSV   | be present for ``PollingLocation`` to be |
-|                  |                                         |              |              | implementations, repeating               | valid. If no valid ``AddressLine`` is    |
-|                  |                                         |              |              | ``address_line`` columns should be       | present, the implementation is required  |
-|                  |                                         |              |              | incremented, e.g. address_line1,         | to ignore the ``PollingLocation``        |
-|                  |                                         |              |              | address_line2, address_line3.            | element containing it.                   |
+|                  |                                         |              |              | address to a polling location.           | be present for ``PollingLocation`` to be |
```
Comment (jdmgoogle) [inline docs/built_rst/tables/elements/polling_location.rst]: Same question about why these were removed.

File: docs/built_rst/xml/elements/polling_location.rst [hunk 3]
```diff
@@ -11,11 +11,11 @@ The PollingLocation object represents a site where voters cast or drop off ballo
...
-|                  |                                         |              |              | address to a polling location. For CSV   | be present for ``PollingLocation`` to be |
-|                  |                                         |              |              | implementations, repeating               | valid. If no valid ``AddressLine`` is    |
-|                  |                                         |              |              | ``address_line`` columns should be       | present, the implementation is required  |
-|                  |                                         |              |              | incremented, e.g. address_line1,         | to ignore the ``PollingLocation``        |
-|                  |                                         |              |              | address_line2, address_line3.            | element containing it.                   |
+|                  |                                         |              |              | address to a polling location.           | be present for ``PollingLocation`` to be |
```
Comment (jdmgoogle) [inline docs/built_rst/xml/elements/polling_location.rst]: Here too.

File: docs/yaml/elements/internationalized_text.yaml [hunk 4]
```diff
@@ -1,34 +1,15 @@
...
-  * :ref:`$$$-party`
-
-  * :ref:`$$$-person`
-
-  * :ref:`$$$-polling-location`
-
-  * :ref:`$$$-source`
-
-  NOTE: Internationalized Text is not currently supported for CSV submissions. 
+description: "``InternationalizedText`` allows for support of multiple languages for\
```
Comment (jdmgoogle) [inline docs/yaml/elements/internationalized_text.yaml]: Is this escaping correct/intentional?

File: docs/yaml/elements/polling_location.yaml [hunk 5]
```diff
@@ -18,9 +18,7 @@ tags:
 - _name: AddressLine
   csv-header-name: address_line
   csv-type: xs:string
-  description: Represents the various parts of an address to a polling location. For
-    CSV implementations, repeating ``address_line`` columns should be incremented,
-    e.g. address_line1, address_line2, address_line3.
+  description: Represents the various parts of an address to a polling location.
```
Comment (jdmgoogle) [inline docs/yaml/elements/polling_location.yaml]: Ah, the root cause of my repeated comments.

File: docs/built_rst/csv/elements/polling_location.rst (hunk 1 above)
Comment (afsmythe) [reply docs/built_rst/csv/elements/polling_location.rst]: The notes were incorrect. CSV implementation does support address_line1, address_line2 fields. Only one address_line is allowed.

File: docs/yaml/elements/internationalized_text.yaml (hunk 4 above)
Comment (afsmythe) [reply docs/yaml/elements/internationalized_text.yaml]: These have been removed.

**Pull request merged**